src/ordinal.js

```diff
@@ -34,6 +36,15 @@ export default function ordinal() {
     return arguments.length ? (unknown = _, scale) : unknown;
   };
 
+  scale.invert = function(_) {
+    if (!invIndex) {
+      invIndex = map();
+      var n = domain.length;
+      while (--n >= 0) invIndex.set(range[n], n+1);
+    }
+    return domain[invIndex.get(_ + "") - 1];
```

Comment:
Coercing `_` to a string shouldn’t be necessary here since it will be done by *map*.get.